docs/iris/example_tests/__init__.py

```diff
@@ -1,19 +1,7 @@
-# (C) British Crown Copyright 2010 - 2015, Met Office
+# Copyright Iris Contributors
```

Comment:
`# Copyright Iris contributors` not `# Copyright Iris Contributors`

Reply:
👍